bidrequest.go

```diff
@@ -16,22 +17,22 @@ var (
-	Imp         []Impression    `json:"imp,omitempty"`
-	Site        *Site           `json:"site,omitempty"`
-	App         *App            `json:"app,omitempty"`
-	Device      *Device         `json:"device,omitempty"`
-	User        *User           `json:"user,omitempty"`
-	Test        int             `json:"test,omitempty"`    // Indicator of test mode in which auctions are not billable, where 0 = live mode, 1 = test mode
-	AuctionType int             `json:"at"`                // Auction type, where 1 = First Price, 2 = Second Price Plus. Exchange-specific auction types can be defined using values greater than 500.
-	TMax        int             `json:"tmax,omitempty"`    // Maximum amount of time in milliseconds to submit a bid
-	WSeat       []string        `json:"wseat,omitempty"`   // Array of buyer seats allowed to bid on this auction
-	AllImps     int             `json:"allimps,omitempty"` // Flag to indicate whether exchange can verify that all impressions offered represent all of the impressions available in context, Default: 0
-	Cur         []string        `json:"cur,omitempty"`     // Array of allowed currencies
-	Bcat        []string        `json:"bcat,omitempty"`    // Blocked Advertiser Categories.
-	BAdv        []string        `json:"badv,omitempty"`    // Array of strings of blocked toplevel domains of advertisers
-	Regs        *Regulations    `json:"regs,omitempty"`
-	Ext         json.RawMessage `json:"ext,omitempty"`
+	ID          string           `json:"id"` // Unique ID of the bid request
+	Imp         []Impression     `json:"imp,omitempty"`
+	Site        Site             `json:"site,omitempty"`
+	App         App              `json:"app,omitempty"`
+	Device      Device           `json:"device,omitempty"`
+	User        User             `json:"user,omitempty"`
+	Test        int              `json:"test,omitempty"`    // Indicator of test mode in which auctions are not billable, where 0 = live mode, 1 = test mode
+	AuctionType int              `json:"at"`                // Auction type, where 1 = First Price, 2 = Second Price Plus. Exchange-specific auction types can be defined using values greater than 500.
+	TMax        int              `json:"tmax,omitempty"`    // Maximum amount of time in milliseconds to submit a bid
+	WSeat       []string         `json:"wseat,omitempty"`   // Array of buyer seats allowed to bid on this auction
+	AllImps     int              `json:"allimps,omitempty"` // Flag to indicate whether exchange can verify that all impressions offered represent all of the impressions available in context, Default: 0
+	Cur         []string         `json:"cur,omitempty"`     // Array of allowed currencies
+	Bcat        []string         `json:"bcat,omitempty"`    // Blocked Advertiser Categories.
+	BAdv        []string         `json:"badv,omitempty"`    // Array of strings of blocked toplevel domains of advertisers
+	Regs        Regulations      `json:"regs,omitempty"`
+	Ext         *json.RawMessage `json:"ext,omitempty"`
```

Comment:
Please revert these `json.RawMessage` -> `*json.RawMessage` changes. We already have a pending commit that will clean them up in the next release.